ui/src/features/scans/scansSchemas.ts

```diff
@@ -215,7 +198,7 @@ export interface AnalysisReport extends ScanHistory {
 
 export interface SbomReport extends ScanHistory {
 	engine_id?: string | null;
-	sbom: SbomComponent[];
+	sbom: any[];
```

Comment:
check the UI code base to ensure that there aren't an instances where we try to reference data within the `sbom` object (such as the `name`, `id`, `license_id`, etc.); if so, additional validation of existence of the fields in those instances may be required